config/enums.cwt

```diff
@@ -19,4 +19,37 @@ enums = {
         male
         female
     }
-}
\ No newline at end of file
+    enum[ai_value] = {
```

Comment:
I'm going to merge these for now, but we'll need to find a proper way to handle this using the `script_value` type I think.